atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactory.java

```diff
@@ -37,9 +42,12 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.base.Optional;
```

Comment:
Mix of question / nit comment.  Is there any reason to avoid Java 8 constructs in Atlas (in this particular case java Optionals).  I know that there was some concern over porting the code back to java 7 after we hit those jdk issues.  Is that still a concern?